pkg/core/container/fmt.go

```diff
@@ -32,5 +35,14 @@ func CheckFormat(c *container.Container) error {
 		return fmt.Errorf("incorrect nonce: %w", err)
 	}
 
+	// check if there are repeated attributes
+	uniqueAttr := make(map[string]struct{})
```

Comment:
```suggestion
	uniqueAttr := make(map[string]struct{}, len(attrs))
```
and reuse `attrs := c.Attributes()` in `for`

Reply:
added